htdocs/main.inc.php

```diff
@@ -570,7 +570,7 @@ function analyseVarsForSqlAndScriptsInjection(&$var, $type)
 			exit;
 		}
 
-		$resultFetchUser=$user->fetch('', $login, '', 1, ($entitytotest > 0 ? $entitytotest : -1));
+		$resultFetchUser=$user->fetch('', $login, '', 1, (!empty($conf->entity) ? $conf->entity : ($entitytotest > 0 ? $entitytotest : -1)));
```

Comment:
Few lines before this, there is a call to checkLoginPassEntity and the entity used into this function must be same than into this $user->fetch
So if you change here, you should change in checkLoginPassEntity too. So best is to change the line 488 $entitytotest	= ...

Reply:
So this means the fix introduce a serious security hole.
If the user loaded is not the same than the user we check the authenthication, there is a big problem. 
So parameters of function used to check authentication must be same than parameters used to load the user (we check authentication for an entity $entity line 510 and if authentication is ok, we load a use in another entity (that may be a different user). So we are able with this change to login on a user account using the password of another user.

Reply:
@eldy 
yes it works, I just released the 7.0.2 version of the module, thank you